doc/source/whatsnew/v1.3.0.rst

```diff
@@ -293,10 +293,11 @@ Groupby/resample/rolling
 
 Reshaping
 ^^^^^^^^^
-
 - Bug in :meth:`DataFrame.unstack` with missing levels led to incorrect index names (:issue:`37510`)
+- Bug in :func:`concat` incorrectly casting to ``object`` dtype in some cases when one or more of the operands is empty (:issue:`38843`)
```

Comment:
can you be more specific as this applies to concat of DataFrames *only* and only on axis=1 (next pass ok)

Reply:
next pass sounds good